src/Components/DropMenu.js

```diff
@@ -72,7 +72,7 @@ export default function DropMenu() {
 
   function sendToProfile(e) {
     handleClose(e);
-    navigate(`/profile/${user.uid}`);
+    user ? navigate(`/profile/${user?.uid}`) : navigate("/login");
```

Comment:
nit: if we know user is non-null due to the ternary, we don't need the null-check later.  This can be just: 
```
user ? navigate(`/profile/${user.uid}`) : navigate("/login")
```